common/plugins/eu.esdihumboldt.hale.common.align/plugin.xml

```diff
@@ -445,6 +445,31 @@
             <parameterBinding></parameterBinding>
          </functionParameter>
       </propertyFunction>
+      <propertyFunction
+            category="eu.esdihumboldt.hale.align.general"
+            icon="icons/augmentation.gif"
+            identifier="eu.esdihumboldt.hale.common.align.collector.assign"
+            name="Assign collected references">
+         <targetProperties>
+            <property
+                  eager="false"
+                  maxOccurrence="-1"
```

Comment:
I think this should be `1`, there can only one schema element be supplied as target.
The `minOccurrence` as well, as w/o a target the function cannot be used.